readme.md

```diff
@@ -0,0 +1,63 @@
+### WordCamp.org
+
+This repository mirrors https://meta.svn.wordpress.org/sites/trunk/wordcamp.org/ and can be used to contribute used Git instead of SVN.
+
+###### Setup 
+
+There are two primary ways to setup this repo for local development.
+
+1. Using VVV. WordCamp.org is available as part of WordPress.org's meta network, and can be setup by following meta-environment setup guide: https://github.com/WordPress/meta-environment/blob/master/docs/install.md
+
+1. If you'd like to only setup WordCamp.org and would like to use Docker, you can use inbuilt docker provisioning. Please follow these steps
```

Comment:
I would think Docker and VVV instructions should reside within same parent folder. Maybe same level pages on make/meta or inside a docs folder in git repo with readme linking to those. What do you think?

Reply:
🤔 Having them inside the repo is probably best, so that people don't have to switch back and forth between a browser and IDE/terminal. Having it in the repo also gives us a rendered version on GitHub.com that we can link to from the handbooks.

I think it'd be good for `/README.md` to _mention_ the VVV and Docker instructions, and link to them, but having the full instructions in the main readme feels overwhelming and cluttered to me. I think it'll make it difficult for people to navigate that document and find what they need.

The VVV instructions should probably remain inside the Meta Environment repo, though, since that's where the provisioning code/database/etc is. It'd be good to mention/link to it from `/README.md`, though.

I like the idea of a `docs/` folder, but it doesn't seem like we have enough for that yet, since right now it's just this file. Maybe as part of #2 we could discuss moving some of our internal devdocs there, though? Even if we did move it there, I still think the best location for the Docker readme would be inside `.docker/`, but it'd definitely be good to link (or just symlink) to that from `docs/`.

I'm open to other ideas, though. Do you have any thoughts, @coreymckrill ?

Reply:
I splitted readme.md in https://github.com/WordPress/wordcamp.org/pull/113/commits/db5db5a1d8c4db4cd3f90b0c97aa60795cfa729a . Please resolve this conversation if this is good enough for now.

Reply:
👍